lib/eu_central_bank.rb

```diff
@@ -209,13 +209,19 @@ def update_parsed_historical_rates(doc)
     @historical_last_updated = Time.now
   end
 
+
+  #new method for round
+  def exchange_with_round(from, to_currency, round = 2, date = nil)
```

Comment:
Thank you for making the changes and sorry for a delay getting back to you. Do you think it's useful to have this method when someone can just call the `.exchange_with().round()` themselves? Seems like this method doesn't bring much added value, but I might be missing something